src/Metrics.ts

```diff
@@ -87,23 +109,23 @@ export class Metrics {
         metrics.counters = {}
 
         //  Preserve any prior defined properties functions
-        metrics.properties = metrics.properties || prior.properties
+        metrics.properties = metrics.properties || prior.metrics.properties
```

Comment:
Not 100% if this change is right. But looking at the call in Table.js, I think prior is a Metrics instance, so I think this was just buggy before.

Reply:
I think you are right.